src/cli/Show.cpp

```diff
@@ -104,6 +104,18 @@ int Show::executeWithDatabase(QSharedPointer<Database> database, QSharedPointer<
         }
     }
 
+    // Separate attributes from attachments via a newline.
+    out << endl;
+    out << QObject::tr("Attachments:") << endl;
```

Comment:
I'm not sure we should print that if there are no attachments. But users might also think that the feature is not implemented at all. Maybe we could print a specific string, like `No attachments`?

Reply:
Thanks for pointing that out or else I'd have probably forgotten it. I was going to get back to you for more specific details regarding the formatting anyway. Printing `No attachments` sounds like a good idea.